profiler/CMakeLists.txt

```
@@ -1,7 +1,10 @@
-
 add_subdirectory(src)
+# add_subdirectory(include)
```

Should we do this instead of installing it below?

```suggestion
add_subdirectory(gz)
```

---

I tried doing it that way at first, but I was getting an error:
```
Developer error: You are missing the file
  [.../ign_ws/src/ign-common/profiler/include/gz/common/config.hh.in]!
```

I suspect it's from the placement of the CMakeLists file or the fact that there's no further components inside. I didn't want to move the folder hierarchy around since there was a baze file as well :X